Assets/HoloToolkit-Examples/MotionControllers-GrabMechanics/Scripts/RotatableObject.cs

```diff
@@ -17,28 +17,35 @@ public class RotatableObject : BaseUsable
         protected override void OnEnable()
         {
             base.OnEnable();
-            InteractionManager.InteractionSourceUpdated += GetTouchPadPosition;
+            //InteractionManager.InteractionSourceUpdated += GetTouchPadPosition;
         }
 
         protected override void OnDisable()
         {
             base.OnDisable();
-            InteractionManager.InteractionSourceUpdated -= GetTouchPadPosition;
+           // InteractionManager.InteractionSourceUpdated -= GetTouchPadPosition;
         }
 
         /// <summary>
         /// In the BaseUsable class that this class inherits from, UseStarted begins checking for usage
         /// after the object is grabbed/picked up
         /// </summary>
         /// <param name="obj"></param>
+        /// 
+
+        protected void Start()
```

Comment:
should this be virtual?

Reply:
I didn't see a Start() method in the latest version of that script, but I did see a similar, non-virtual method for Awake() with no content. I've deleted the empty function call.